pytket/docs/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+1.31.1 (August 2024)
+--------------------
+
+* Fix issue in pytket-circuit-renderer 0.9.
```

Comment:
Fix issue with css styles in pytket-circuit-renderer 0.9. ?

Reply:
Not sure if we need that much details, happy to update that if you want to.

Reply:
Up to you, its a minor change. I'm fine with either.

Reply:
```suggestion
* Fix issue with css styles in pytket-circuit-renderer 0.9.